Formula/gjs.rb

```diff
@@ -1,8 +1,8 @@
 class Gjs < Formula
   desc "JavaScript Bindings for GNOME"
   homepage "https://wiki.gnome.org/Projects/Gjs"
```

Comment:
The homepage has moved. Should probably be `https://gitlab.gnome.org/GNOME/gjs/wikis/Home`